libs/core/src/components/pds-box/pds-box.tsx

```diff
@@ -182,35 +182,102 @@ export class PdsBox {
   @Prop() shadow?: BoxShadowSizeType;
 
   /**
-   * Size of the column for all screen sizes that are not explicitly set.
+   * Sets the default column width for the component. This value applies from the smallest screen size (XS) upwards, unless overridden by a breakpoint-specific size prop at that breakpoint or larger.
    */
   @Prop() size?: BoxColumnType;
 
   /**
-   * At screen sizes less than the `XS` breakpoint, this will take the number of columns specified.
+   * At the XS breakpoint (typically the smallest screen sizes, 0px to 575px), this will take the number of columns specified. This overrides the base size prop for this specific range.
    */
   @Prop() sizeXs?: BoxColumnType;
 
   /**
-   * At screen sizes greater than the `SM` breakpoint, this will take the number of columns specified.
+   * At screen sizes from the SM breakpoint and larger (576px and up), this will take the specified number of columns. This overrides any value set by size or sizeXs.
    */
   @Prop() sizeSm?: BoxColumnType;
 
   /**
-   * At screen sizes greater than the `MD` breakpoint, this will take the number of columns specified.
+   * At screen sizes from the MD breakpoint and larger (768px and up), this will take the specified number of columns. This overrides any value set by size, sizeXs, or sizeSm.
```

Comment:
Make this specific change please:
```suggestion
   * At screen sizes from the MD breakpoint and larger (768px and up), this will take the specified number of columns. This overrides any value set by size, sizeXs, and up.
```